### ISSUE TITLE
noConflict method not working

### ISSUE THREAD

**Issue by amozes** [mapbox/mapbox.js#1087]
When using the documented noConflict method, it fails.
The error reported in console is `L is undefined`.

See fiddle: http://jsfiddle.net/dhbtdsq7/

Please let me know if I'm doing something wrong and/or misinterpreted the doc.

**Comment by jfirebaugh** [comment]
Yes, it looks like Mapbox.js is not compatible with `L.noConflict()`. It should be possible to make it compatible, but will require some internal work -- namely, not relying on the global `L` except at script load time.
